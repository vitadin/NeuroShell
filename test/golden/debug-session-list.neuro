%% Debug test: create one session and list
\set[_echo_command="true"]
\session-new debug_session
\session-list