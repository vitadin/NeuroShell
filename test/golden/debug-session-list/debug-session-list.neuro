%% Debug test: create one session and list
\set[_echo_commands="true"]
\session-new debug_session
\session-list