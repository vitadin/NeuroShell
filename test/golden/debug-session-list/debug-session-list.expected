Setting _echo_commands = true
%%> \session-new debug_session
Created session 'debug_session' (ID: 00000001)
%%> \session-list
Sessions (1 total):
  debug_session    (ID: 00000001, active, 0 messages, created: 2025-01-01)